src/table/internal.tsx

```diff
@@ -119,6 +119,14 @@ const InternalTable = React.forwardRef(
     const [tableWidth, tableMeasureRef] = useContainerQuery<number>(rect => rect.contentBoxWidth);
     const tableRefObject = useRef(null);
 
+    const containerContentWidth = useMemo(() => {
```

Comment:
Ideally is to avoid these computations in the code. In theory, we can apply the paddings to the container itself in which case its measured content width is going to be correct. However, that requires complex adjustments applied to other styles and also resizable- and sticky columns features.